tests/Provider/Node/SystemNodeProviderTest.php

```diff
@@ -566,10 +597,10 @@ private function assertMockFunctions(
                 $this->functionProxies[$key]->verifyNeverInvoked();
             } elseif (is_array($asserts)) {
                 foreach ($asserts as $assert) {
-                    $this->functionProxies[$key]->verifyInvokedOnce($assert);
+                    $this->functionProxies[$key]->verifyInvoked($assert);
```

Comment:
I am not to sure about this part. It feels hacky to either make a different flow just for `ini_get` or make it like this with just checking if it was invoked without checking how often it was invoked.

@ramsey How would you proceed?